test/checks/label/implicit.js

```diff
@@ -2,33 +2,34 @@ describe('implicit-label', function () {
 	'use strict';
 
 	var fixture = document.getElementById('fixture');
+	var fixtureSetup = axe.testUtils.fixtureSetup;
 
 	afterEach(function () {
 		fixture.innerHTML = '';
 	});
 
 	it('should return false if an empty label is present', function () {
-		fixture.innerHTML = '<label><input type="text" id="target"></label>';
+		fixtureSetup('<label><input type="text" id="target"></label>');
 		var node = fixture.querySelector('#target');
 		assert.isFalse(checks['implicit-label'].evaluate(node));
 	});
 
 	it('should return false if an invisible non-empty label is present', function () {
-		fixture.innerHTML = '<label><span style="display: none">Text</span> <input type="text" id="target"></label>';
+		fixtureSetup('<label><span style="display: none">Text</span> <input type="text" id="target"></label>');
 		var node = fixture.querySelector('#target');
 		assert.isFalse(checks['implicit-label'].evaluate(node));
 	});
 
 	it('should return true if a non-empty label is present', function () {
-		fixture.innerHTML = '<label>Text <input type="text" id="target"></label>';
+		fixtureSetup('<label>Text <input type="text" id="target"></label>');
 		var node = fixture.querySelector('#target');
 		assert.isTrue(checks['implicit-label'].evaluate(node));
 	});
 
 	it('should return false if a label is not present', function () {
 		var node = document.createElement('input');
 		node.type = 'text';
-		fixture.appendChild(node);
+		fixtureSetup(node);
 
 		assert.isFalse(checks['implicit-label'].evaluate(node));
 	});
```

Comment:
This needs to support this example

```
        <form>
            <label>Option 1<div class="slotted"><input type="text" name="slottedLabel" /></div></label>
        </form>
        <script>
            function createContentSlotted() {
                var group = document.createElement('p');
                group.innerHTML = '<slot></slot>';
                return group;
            }

            function makeShadowTreeSlotted(node) {
                var root = node.attachShadow({mode: 'open'});
                root.appendChild(createContentSlotted());
            }

            document.addEventListener('DOMContentLoaded', function() {
                document.querySelectorAll('.slotted').forEach(makeShadowTreeSlotted);
            });
        </script>
```

Which works properly in both Safari and Chrome with VO